module/actor/sheet/actor-sheet.js

```diff
@@ -29,9 +29,58 @@ export default class PBActorSheet extends (foundry.appv1?.sheets?.ActorSheet ??
   constructEffectLists(sheetData) {
     const effects = {};
 
-    effects.temporary = sheetData.actor.effects.filter((i) => i.isTemporary && !i.disabled && !i.isCondition);
-    effects.disabled = sheetData.actor.effects.filter((i) => i.disabled && !i.isCondition);
-    effects.passive = sheetData.actor.effects.filter((i) => !i.isTemporary && !i.disabled && !i.isCondition);
+    // Helper function to add modifier display to effects
+    const addModifierDisplay = (effect) => {
+      const modifiers = [];
+      effect.changes.forEach(change => {
+        if (change.key.includes('combat.')) {
+          const modifier = change.key.replace('system.attributes.combat.', '');
+          const value = change.value;
+          let displayName = '';
+          
+          switch (modifier) {
+            case 'attackModifier': displayName = 'Attack'; break;
+            case 'defenseModifier': displayName = 'Defense'; break;
+            case 'initiativeModifier': displayName = 'Initiative'; break;
+            case 'damageModifier': displayName = 'Damage'; break;
+            case 'armorTierModifier': displayName = 'Armor Tier'; break;
+            case 'speedModifier': displayName = 'Speed'; break;
+            case 'luckDieModifier': displayName = 'Luck Die'; break;
```

Comment:
Default case, split on either `combat.` or `system.attributes.combat.` and show the portion after it?